docs/tutorials/using-the-OFF-API-tutorial.md

```diff
@@ -106,4 +106,111 @@ The `product` object in the response now contains the extra fields to show how t
+
+<!-- I dont know if using 100% Real Orange Juice is a good approach for now , should we state that it was not computed at the time of writing this article just incase it gets computed in future or there is a product we can use to test this that wont change in future ? -->
+
+To know the missing tags, you need the `misc-tags` field from the response.
+
+`https://world.openfoodfacts.org/api/v2/product/0180411000803/100-real-orange-juice?fields=misc_tags`
+
+The response shows the missing fields and category needed to compute the Nutri-Score.
+
+```json
+{
+    "code": "0180411000803",
+    "product": {
+        "misc_tags": [
+            "en:nutriscore-not-computed",
+            "en:nutriscore-missing-category",
+            "en:nutrition-not-enough-data-to-compute-nutrition-score",
+            "en:nutriscore-missing-nutrition-data",
+            "en:nutriscore-missing-nutrition-data-sodium",
+            "en:ecoscore-extended-data-not-computed",
+            "en:ecoscore-not-computed",
+            "en:main-countries-new-product"
+        ]
+    },
+    "status": 1,
+    "status_verbose": "product found"
+}
+```
+
+The sample response above for 100% Real Orange Juice `misc_tags` shows that the Nutri-Score is missing category and sodium(salt). Now you can write to the OFF API to provide these nutriment data (if you have them) so that the Nutri-Score can be computed.
+
+### Write data to make Nutri-Score computation possible
+
+The WRITE operations in the OFF API require authentication, therefore you need a valid `user_id` and `password`  to write the missing nutriment data to 100% Real Orange Juice. 
+
+> Sign up on the [Open Food Facts App](https://world.openfoodfacts.org/), to get your `user_id` and `password` if you dont have.
+
+To write data to a product, make a `POST` request to the [`Add or Edit A Product`](https://openfoodfacts.github.io/openfoodfacts-server/reference/api.html#tag/Write-Requests/operation/post-cgi-product_jqm2.pl) endpoint.
+
+```bash
+https://world.openfoodfacts.org/cgi/product_jqm2.pl
+```
+
+For authentication, add your valid `user_id` and `password` as body parameters to your request. The `code` (barcode of the product to be added/edited), `user_id` and `password` are required fields when adding or editing a product. Then, include other product data to be added in the request body.
+
+To write `sodium` and `category` to 100% Real Orange Juice so that the Nutri-Score can be computed, the request body should contain these fields :
+
+| Key        | Value           | Description  |
+| ------------- |:-------------:| -----:|
+| user_id     | *** | A valid user_id |
+| password      | ***     |   A valid password |
+| code | 0180411000803      |    The barcode of the product to be added/edited |
+| nutriment_sodium | 0.015      |    Amount of sodium |
+| nutriment_sodium_unit | g      |   Unit of sodium relative to the amount |
+| categories | Orange Juice     |   Category of the Product |
+
```

Comment:
I would suggest adding how it looks with curl. 

Here (using .net):
````suggestion

Using curl:
```bash
curl -XPOST -u off:off -x POST https://world.openfoodfacts.org/cgi/product_jqm2.pl \
  -F user_id=your_user_id -F password=your_password \
  -F code=0180411000803 -F nutriment_sodium=0.015 -F nutriment_sodium_unit=g -F categories="Orange Juice"
```

````

Note that the firefox dev console (maybe also the chrome one), in network tab, enables you to get the curl for an issued request.

Reply:
Your suggestion doesn't use .net, do I update it and secondly as I mentioned earlier  : https://github.com/openfoodfacts/openfoodfacts-server/pull/7640#discussion_r1012804627

Reply:
updated to use .net ;-) (this was a typo)